Brokerages/Backtesting/BacktestingBrokerage.cs

```diff
@@ -353,6 +353,20 @@ public void Scan()
                                     fills = option.OptionExerciseModel.OptionExercise(option, order as OptionExerciseOrder).ToArray();
                                     break;
                             }
+
+                            // invoke fee models for completely filled order events
+                            foreach (var fill in fills)
+                            {
+                                if (fill.Status == OrderStatus.Filled)
+                                {
+                                    // this check is provided for backwards compatibility of older user-defined fill models
+                                    // that may be performing fee computation inside the fill model w/out invoking the fee model
+                                    if (fill.OrderFee == 0m)
```

Comment:
What are your thoughts about checking whether or not `fill` is the new base class `FillModel` or a derivative of it, versus checking for order fee 0? Maybe using `IsAssignableFrom`

Reply:
great idea, since it's definitely possible that a custom `IFillModel` implemented zero fees explicitly and in contradiction to the configured `IFeeModel` -- I'll make this change 👍

Reply:
after more thought, I'm not sure this will be great as it would cause a bug if a user directly implements the `IFillModel` interface, for example, in python. Now, we could also check for the python type (which doesn't subclass `FillModel`), but I think for now checking zero is sufficient. I'll add a note that we should remove the check in 6 months